docs/runbooks/SSPHighRateRejectedVms.md

```diff
@@ -1,25 +1,33 @@
 # SSPHighRateRejectedVms
+<!--apinnick Nov 2022-->
 
 ## Meaning
 
-This alert indicates that many Virtual Machines were not valid and therefore were not created or not modified.
-It means either a user or a script is continuously trying to create invalid Virtual Machines.
+This alert fires when a large number of invalid virtual machines (VMs) is detected by the Template Validator.
 
 ## Impact
 
-There is no immediate impact, however one should see what is the reason for the large amount of rejected Virtual Machines.
-A Large amount of Virtual Machines may mean that something is not going to work as planned, for example, a script may be consistently failing to create or modify Virtual Machines
+There is no immediate impact. However, you should determine the cause, such as a user or script continuously creating invalid VMs.
```

Comment:
I believe we are missing the part here that it means that these VMs will not be created or modified and this may mean that things will not work as planned.

Reply:
If that is the case, then I think the "There is no immediate impact" sentence should be removed.

Reply:
@apinnick I agree.